crates/protocol/derive/src/traits/pipeline.rs

```diff
@@ -1,14 +1,13 @@
 //! Defines the interface for the core derivation pipeline.
 
+use crate::{errors::PipelineErrorKind, traits::OriginProvider, types::StepResult};
 use alloc::boxed::Box;
 use async_trait::async_trait;
 use core::iter::Iterator;
 use kona_genesis::{RollupConfig, SystemConfig};
 use kona_protocol::L2BlockInfo;
 use kona_rpc::OpAttributesWithParent;
 
-use crate::{errors::PipelineErrorKind, traits::OriginProvider, types::StepResult};
-
```

Comment:
Much prefer the previous formatting personally. Clear separation of external vs local types.